src/bulk/common.ts

```diff
@@ -1180,7 +1181,10 @@ export abstract class BulkOperationBase {
     options = options ?? {};
 
     if (this.s.executed) {
-      return handleEarlyError(new MongoDriverError('Batch cannot be re-executed'), callback);
+      return handleEarlyError(
+        new MongoBatchReExecutionError('Batch cannot be re-executed'),
```

Comment:
Along the lines of Daria's comment: I think this is a good place to apply the 'what must be' style of message like, 'A new bulk operation must be created to in order to execute' as a rough suggestion, improvement needed of course

Reply:
Would something like this be appropriate:

`'This batch has already been executed: Must create new batch to execute'`

Reply:
'This batch has already been executed, create new batch to execute' maybe?